openfisca_france/model/prelevements_obligatoires/taxe_habitation.py

```diff
@@ -64,14 +64,14 @@ class taxe_habitation(Variable):
     url = "http://www.impots.gouv.fr/portal/dgi/public/particuliers.impot?espId=1&pageId=part_taxe_habitation&impot=TH&sfid=50"
 
     def function(self, simulation, period):
-        period = period.start.offset('first-of', 'month').period('year')
+        period = period.start.offset('first-of', 'month').period(-1,'year')
```

Comment:
This should not be changed since it enters many variable calculations and we need it only for rfr_n_1 (rfr_last_year)